pkg/installer/custominstallconfig.go

```diff
@@ -54,16 +56,30 @@ func (m *manager) applyInstallConfigCustomisations(installConfig *installconfig.
 	}
 
 	aroManifests := &AROManifests{}
-	fileFetcher := &aroFileFetcher{directory: "/"}
-	aroManifestsExist, err := aroManifests.Load(fileFetcher)
+	manifestsFileFetcher := &aroFileFetcher{directory: "/"}
+	aroManifestsExist, err := aroManifests.Load(manifestsFileFetcher)
 	if err != nil {
-		m.log.Errorf("Error loading ARO manifests: %v", err)
+		err = fmt.Errorf("error loading ARO manifests: %w", err)
+		m.log.Error(err)
+		return nil, err
+	}
+
+	boundSaSigningKey := &tls.BoundSASigningKey{}
+	boundSaSigningKeyFileFetcher := &aroFileFetcher{directory: "."}
```

Comment:
We can probably use the same fileFetcher as for manifests here?

Reply:
I made a different one, because based on my investigation of the upstream installer code as well as Hive, the directory being read from is `./tls/` - whatever `.` ends up being, I chose to keep this in-line with upstream.

Reply:
Ohh, tls and manifests are supposed to be in the same directory, for us `.` ends up being `/`. That's why we also use the same for `.azure` and `manifests`

Reply:
Should we update the manifests code to read from `.` instead of `/`? Does the working directory of the installer ultimately just end up being `/` anyways?

Reply:
Decided to standardize both of these fileFetchers to read the `--dir` flag (which defaults to `.`)